includes/admin/helper/views/html-oauth-start.php

```diff
@@ -19,7 +19,7 @@
 				<img src="<?php echo esc_url( WC()->plugin_url() . '/assets/images/woocommerce_logo.png' ); ?>" alt="WooCommerce" style="width:180px;">
 
 				<?php if ( ! empty( $_GET['wc-helper-status'] ) && 'helper-disconnected' === $_GET['wc-helper-status'] ) : ?>
-					<p><strong><?php esc_html_e( 'Sorry to see you go.', 'woocommerce' ) . '</strong>' . esc_html_e( 'Feel free to reconnect again using the button below.', 'woocommerce' ); ?></p>
+					<p><strong><?php esc_html_e( 'Sorry to see you go.', 'woocommerce' ) ?></strong> <?php esc_html_e( 'Feel free to reconnect again using the button below.', 'woocommerce' ); ?></p>
```

Comment:
You are missing a semicolon after the first esc_html_e

Reply:
Fixed